packages/launch/src/focused-launch/summary/index.tsx

```diff
@@ -288,35 +288,46 @@ const PlanStep: React.FunctionComponent< PlanStepProps > = ( {
 		select( LAUNCH_STORE ).getSelectedPlanProductId()
 	);
 
-	const selectedPaidPlanProductId = useSelect( ( select ) =>
-		select( LAUNCH_STORE ).getPaidPlanProductId()
-	);
-
-	const { selectedPlan, selectedPaidPlan, billingPeriod } = useSelect( ( select ) => {
+	const { selectedPlan, selectedPlanProduct } = useSelect( ( select ) => {
 		const plansStore = select( PLANS_STORE );
 
 		return {
 			selectedPlan: plansStore.getPlanByProductId( selectedPlanProductId, locale ),
-			selectedPaidPlan: plansStore.getPlanByProductId( selectedPaidPlanProductId, locale ),
-			billingPeriod:
-				plansStore.getPlanProductById( selectedPlanProductId )?.billingPeriod || 'ANNUALLY',
+			selectedPlanProduct: plansStore.getPlanProductById( selectedPlanProductId ),
 		};
 	} );
 
-	const { defaultPaidPlan, defaultFreePlan } = usePlans();
-
 	// persist non-default selected paid plan if it's paid in order to keep displaying it in the plan picker
 	const [ nonDefaultPaidPlan, setNonDefaultPaidPlan ] = React.useState< Plan | undefined >();
+	const [ nonDefaultPaidPlanProduct, setNonDefaultPaidPlanProduct ] = React.useState<
+		PlanProduct | undefined
+	>();
+
+	const [ billingPeriod, setBillingPeriod ] = React.useState< PlanProduct[ 'billingPeriod' ] >(
```

Comment:
Another nit: By adding this import to the file:

```typescript
import type { Plans } from '@automattic/data-stores';
```

we can then use the `PlanBillingPeriod` type:


```suggestion
	const [ billingPeriod, setBillingPeriod ] = React.useState< Plans.PlanBillingPeriod >(
```

Reply:
This code is gone now ✨